Project/SideMenuController/SideMenuController/SideMenuController.swift

```diff
@@ -272,6 +272,11 @@ class SideMenuController: UIViewController, UIGestureRecognizerDelegate {
     
     // MARK:- Containment -
     
+    func ifSidePanelVisible() -> Bool{
```

Comment:
- this method name doesn't make sense in english
- secondly, why do you need a method when you can do:  `private(set) public var sidePanelVisible ` 

Thank you for appreciating `SideMenuController` but I'll have to close this PR. I'll update the control soon to include all the required features.